config.example.wikimedia.yaml

```diff
@@ -27,6 +27,8 @@ default_project: &default_project
           cache-control: s-maxage=864000, max-age=86400
         mobileapps:
           host: http://appservice.wmflabs.org
+        # 10 days Varnish caching, one day client-side
+        purged_cache_control: s-maxage=864000, max-age=86400
```

Comment:
Yeah, added purging and longer-term caching for that in ec00898138d331c687d11c1f6447c332950117e0.